### ISSUE TITLE
Tox support

### ISSUE THREAD

**Issue by orsinium** [orsinium/dephell#2]

**Comment by michaelaye** [comment]
I see that there was a merge related to this, so is `tox` supported now?

**Comment by orsinium** [comment]
No, it's not :( The PR is about totally different things, it was referenced by a typo.

**Comment by agritheory** [comment]
@orsinium What would it take to make `[tool.dephell.tox]` a feature?
Alternatively, its cousin [nox](https://nox.thea.codes/en/stable/) might be easier to integrate and would offer very similar funcationality

**Comment by orsinium** [comment]
Could you share your use case why you need it and what you expect from this integration? The issue is only about reading dependencies that are specified in the tox config itself. It's not hard to implement since it's just about reading ini-file. If you're looking for a way to run tests for many environments with dephell.org , `dephell project test` and `dephell generate travis` should help.

**Comment by graingert** [comment]
>  It's not hard to implement since it's just about reading ini-file. 

tox is actually quite hard to parse, because it's got [factors and factor conditional settings](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings) and complex factor expressions, factor value substitutions, inheritance and environment variable substitution
